frontend/src/components/program/ScheduleEntryFilters.vue

```diff
@@ -223,6 +223,12 @@ export default {
       this.loadEndpointData('campCollaborations', 'responsible', true)
       this.loadEndpointData('progressLabels', 'progressLabel', true)
     }
+    const resizeObserver = new ResizeObserver(() => {
+      const rs = getComputedStyle(this.$el)
+      const h = rs.getPropertyValue('height')
+      this.$emit('height-changed', h)
+    })
+    resizeObserver.observe(this.$el)
```

Comment:
Why don't you use the `v-resize-observer` directive we already have? Otherwise you need to discard the observer on unmount. Also wouldn't it make more sense to only use it for desktop inside CampProgram.vue instead of ScheduleEntryFilters.vue?

Reply:
cool - changed it :)